internal/metrics/listener.go

```diff
@@ -148,16 +150,51 @@ func getEnhancedMetricsTags(ctx context.Context) []string {
 	isColdStart := ctx.Value("cold_start")
 
 	if lc, ok := lambdacontext.FromContext(ctx); ok {
-		// ex: arn:aws:lambda:us-east-1:123497558138:function:golang-layer
+		// ex: aS
 		splitArn := strings.Split(lc.InvokedFunctionArn, ":")
+		var alias string
+		var executedversion string
```

Comment:
Nit: for this local variable, convention would be to use camelcase. e.g. `executedVersion`.